.github/ISSUE_TEMPLATE/new-repo-checklist.md

```diff
@@ -62,10 +64,10 @@ https://github.com/maplibre/[repo]
   *Use the gear icon in the upper right corner of the repo page.*
 - [ ] CI automatically runs on all pull requests before merging using GitHub actions
 - [ ] Grant admin rights to the board members and automation accounts for packages <list-of-packages>
-  - [npmjs.com](https://www.npmjs.com/): package settings / invite:  `maplibreorg nyurik klokan lseelenbinder wipfli`
-  - [crates.io](https://crates.io/): package settings / add owner: `nyurik klokan lseelenbinder wipfli`
+    - [npmjs.com](https://www.npmjs.com/): package settings / invite:  `maplibreorg nyurik klokan lseelenbinder wipfli`
```

Comment:
```suggestion
    - [npmjs.com](https://www.npmjs.com/): package settings / invite:  `maplibreorg nyurik klokan lseelenbinder`
```

Reply:
@birkskyum what's your NPM account?  please add ^

Reply:
It's "birkskyum"